ovs/lib/storagerouter.py

```diff
@@ -810,10 +822,10 @@ def add_vpool(parameters):
             ServiceManager.start_service(alba_proxy_service, client=root_client)
 
         ServiceManager.add_service(name='ovs-volumedriver', params=sd_params, client=root_client, target_name=sd_service)
+        ServiceManager.start_service(sd_service, client=root_client)
 
         storagedriver = StorageDriver(storagedriver.guid)
         current_startup_counter = storagedriver.startup_counter
-        ServiceManager.start_service(sd_service, client=root_client)
```

Comment:
The order here is important. In theory this would allow a race condition where the counter is already incremented before the initial value was loaded.